app/fiori-apps.html

```diff
@@ -13,8 +13,8 @@
             };
         </script>
 
-        <script id="sap-ushell-bootstrap" src="https://sapui5.hana.ondemand.com/1.108.27/test-resources/sap/ushell/bootstrap/sandbox.js"></script>
-        <script id="sap-ui-bootstrap" src="https://sapui5.hana.ondemand.com/1.108.27/resources/sap-ui-core.js" data-sap-ui-libs="sap.m, sap.ushell, sap.collaboration, sap.ui.layout" data-sap-ui-compatVersion="edge" data-sap-ui-theme="sap_horizon" data-sap-ui-frameOptions="allow" data-sap-ui-async="true"></script>
+        <script id="sap-ushell-bootstrap" src="https://sapui5.hana.ondemand.com/1.108.42/test-resources/sap/ushell/bootstrap/sandbox.js"></script>
```

Comment:
I recommend that we use an [evergreen version](https://sapui5.hana.ondemand.com/#/topic/2d3eb2f322ea4a82983c1c62a33ec4ae?q=evergreen) here, so that we don't have to update the version over and over again

```suggestion
        <script id="sap-ushell-bootstrap" src="https://sapui5.hana.ondemand.com/1.108/test-resources/sap/ushell/bootstrap/sandbox.js"></script>
```